src/App.tsx

```diff
@@ -14,49 +15,126 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+enum SortType {
+  LENGTH = 'length',
+  NAME = 'name',
+  EMTY = '',
```

Comment:
```suggestion
  EMPTY = 'empty',
```
typo and better to set some value instead empty string.